azure-pipelines.yml

```diff
@@ -395,13 +395,14 @@ jobs:
   timeoutInMinutes: 30
   dependsOn:
     - chip_earlgrey_cw310
+    - chip_earlgrey_cw310_splice_mask_rom
     - sw_build
   steps:
   - template: ci/install-package-dependencies.yml
   - template: ci/download-artifacts-template.yml
     parameters:
       downloadPartialBuildBinFrom:
-        - chip_earlgrey_cw310
```

Comment:
Maybe: We may need both here, I'm not sure if it's one bin-dir or a number of partial artifacts.